CrowdFundingApplication/Views/Project/CreateProject.cshtml

```diff
@@ -168,12 +168,99 @@
+                           </div>
+
+                           <div class="row">
+                               <div class="col check1">
+                                    <span asp-validation-for="Bank" class="text-danger"></span>
+                               </div>
+                           </div>
+                           <div class="row mb-3">
+                                <div class="col">
+                                    <div class="input-group">
+                                        <span class="input-group-text wdt" id="basic-addon1">Банк получателя</span>
+                                        <input asp-for="Bank" type="text" class="form-control">
+                                    </div>
+                                </div>
+                           </div>
+
+                            <hr />
+
+                           <div class="row">
+                                <div class="col">
+                                    <div class="input-group">
+                                        <span class="wdt input-group-text">Тип</span>
+                                        <select asp-for="Type" class="form-select w-auto" aria-label="Default select example" asp-items="Html.GetEnumSelectList<OrganizationEnum>()" ></select>
+                                    </div>
+                                </div>
+                           </div>
+
+                            <hr />
+
+                           <div class="row">
+                               <div class="col check1">
+                                    <span asp-validation-for="Initials" class="text-danger"></span>
+                               </div>
+                           </div>
+                           <div class="row mb-3">
+                                <div class="col">
+                                    <div class="input-group">
+                                        <span class="wdt input-group-text">Инициалы для подписи в договоре</span>
+                                        <input asp-for="Initials" type="text" class="form-control" placeholder="И.И.Иванов">
+                                    </div>
+                                </div>
+                           </div>
+
+                            <p class="ml-1 info">*эти данные необходимы для отображение организатора сбора средств.</p>
+
+                    </div>
+                </div>
+            </div>
+        </div>
+
         <div class=" mt-3">
             <div class="row">
                 <div class="col"></div>
                 <div class="col-lg-2"></div>
-                <div class="col-lg-2">
-                    <button class="btn btn-primary btn-block justify-content-end but_continue" type="submit">Продолжить</button>
+                <div class="col-lg-4">
+                    <button class="btn btn-primary btn-block justify-content-end but_continue" onclick="message();" type="submit">Отправить на модерацию</button>
```

Comment:
Move binding onclick="message() to js code

Reply:
Has been changed